fcentricprospections.client/src/App.tsx

```diff
@@ -1,56 +1,66 @@
-import { useEffect, useState } from 'react';
-import './App.css';
-
-interface Forecast {
-    date: string;
-    temperatureC: number;
-    temperatureF: number;
-    summary: string;
+import { createBrowserRouter, Outlet, RouterProvider, useParams } from 'react-router-dom';
+import { Homepage} from './pages/homePage';
+import { ShopPage} from './pages/shopIdPage';
+import { DataProvider } from './contexts/dataContext';
+const Root = () => {
+    return (
+        <>
+        <Outlet></Outlet>
+        </>
+    );
+}
```

Comment:
I would make sure header and footer are common parts, so there is unity across the app.

So, for example:

```suggestion
const Root = () => {
    return (
        <>
            <header>
                // header items
            </header>

            <Outlet />

            <footer>
                // footer items
            </footer>
        </>
    )

}
```